### PR TITLE
azure: do not skip verifying signatures of downloaded archives

### PR DESCRIPTION
**Pull request by rainers**
Let's see if the signatures can be verified on azure...

### CODE REVIEW

**Comment by rainers** [comment]
Skipping verification of lld and mingw now as no valid signatures are available, in the hope of unblocking nightliy/release builds.

File: .azure-pipelines/build-winrelease.yml [hunk 1]
```diff
@@ -26,6 +26,9 @@ steps:
     displayName: Unzipping docs
   - script: |
       @echo on
+      gpg --import create_dmd_release/codesign/dlang.pgp
+      gpg --import create_dmd_release/codesign/nowak.pgp
+      gpg --import create_dmd_release/codesign/wilzbach.pgp
```
Comment (wilzbach) [inline .azure-pipelines/build-winrelease.yml]: Uhm you can download them as a keyring: https://dlang.org/gpg_keys.html

That's how the install.sh works as well.

Comment (rainers) [reply .azure-pipelines/build-winrelease.yml]: Downloading them from external sources seems to somewhat defeat the purpose of verifying the archives. Is it standard to do this?
I tried `--receive-keys` initially but that caused an I/O error on the keyserver.

Comment (rainers) [reply .azure-pipelines/build-winrelease.yml]: Or do you mean that the keyring file should be added to this repo?

Comment (wilzbach) [reply .azure-pipelines/build-winrelease.yml]: Yes, probably a better idea then though as I mentioned earlier I think sha256 hashes might be easier as then everyone with write access can upload files and not only Martin and myself.

**Comment by rainers** [comment]
Now switched to sha256 for newer archives, but downloading the keyring file for simplicity (azure doesn't build official releases anyway).

**Pull request merged**